corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
Why are we putting this in a separate file rather than settings.py? I think our settings.py is a disaster (it's very complicated to override nested settings locally), but it seems like adding more config files may not be a better solution. With this, if I want to change a database password, for example, I have to add `DATABASE_CONFIG_FILE` in localsettings.py *and* create a new file (and maybe update a .gitignore?).

Any particular reason this is not just a dict in settings.py?

Long term I would love to see hq settings be a yaml file with reasonable defaults (so your localsettings.yml doesn't need to be 100 lines long) with well-designed logic behind it to configure django. Something along the lines of [this SO answer](http://stackoverflow.com/a/18731139) using yaml instead of ini format.